packages/@aws-cdk/aws-appmesh/test/virtual-node.test.ts

```diff
@@ -980,6 +982,69 @@ describe('virtual node', () => {
           },
         });
       });
+
+      test('has an IP Preference applied', () => {
```

Comment:
we're missing a test that verifies that the ip preference is not set by default.

Reply:
There is another test which implicitly tests this, but I've gone ahead and added an explicit test.